index.html

```diff
@@ -8222,6 +8224,548 @@
                 </pre>
     </section>
   </section>
+  <section id="quic-transport*">
+    <h2>The RTCQuicTransport Object</h2>
+    <p>The <dfn><code>RTCQuicTransport</code></dfn> object includes information relating
+    to QUIC transport.</p>
+    <section id="rtcquictransport-overview*">
+      <h3>Overview</h3>
+    </section>
+    <section id="rtcquictransport-operation*">
+      <h3>Operation</h3>
+      <p>A <code><a>RTCQuicTransport</a></code> instance is constructed
+      using an <code><a>RTCIceTransport</a></code> and a sequence of
+      <code><a>RTCCertificate</a></code> objects.
+      An <code><a>RTCQuicTransport</a></code> object in the <code>closed</code> or
+      <code>failed</code> states can be garbage-collected when it is no longer
+      referenced.</p>
+      <p>If an attempt is made to construct a <code><a>RTCQuicTransport</a></code>
+      instance from an <code><a>RTCIceTransport</a></code> in the <code>closed</code>
+      state, an <code>InvalidStateError</code> exception is thrown.
+      If <code>certificates</code> is non-empty,
+      check that the <code>expires</code> attribute of each
+      <code><a>RTCCertificate</a></code> object is in the future.
+      If a certificate has expired, throw an <code>InvalidParameter</code>
+      exception; otherwise, store the certificates.</p>
+      <p>Since the QUIC negotiation occurs between transport endpoints
+      determined via ICE, implementations of this specification <em class="rfc2119"
+      title="MUST">MUST</em> support multiplexing of STUN, TURN, QUIC, DTLS and RTP and/or
+      RTCP.</p>
```

Comment:
Could we leave this part off until we determine how we will demux?  

By the way, I checked Chrome's code, and it it uses the remote address to demux STUN and TURN, and it uses the ICE magic cookie and fingerprint for ICE.   It only looks at the first byte for DTLS and RTP.    So maybe saying it can be demuxed with STUN and TURN, since that seems so easy.  

Along with that, do we really even need demux with DTLS?